src/Superpower/Superpower.csproj

```diff
@@ -25,4 +25,7 @@
   <ItemGroup>
     <None Include="..\..\asset\Superpower-White-400px.png" Pack="true" Visible="false" PackagePath="" />
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="System.Memory" Version="4.6.3" />
```

Comment:
Could we please also keep the dependency conditional on `netstandard2.0`, as in the earlier PR? Thanks!